raw-spans-grouper/raw-spans-grouper/src/test/resources/configs/raw-spans-grouper/application.conf

```diff
@@ -28,6 +28,7 @@ span.window.store.retention.time.mins = ${?SPAN_WINDOW_STORE_RETENTION_TIME_MINS
 span.window.store.segment.size.mins = 20
 span.window.store.segment.size.mins = ${?SPAN_WINDOW_STORE_SEGMENT_SIZE_MINS}
 
+upper.max.span.count = 6
```

Comment:
I think we should call this `default.max.span.count` and the below section can be used as an override per tenant.

Reply:
This makes sense. Made changes.